test/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+	"extends": [
+		"wikimedia/mocha"
```

Comment:
I didn't know we *have* a mocha ESlint declaration! Is that new?

Reply:
Yeah, about 2 months. It's also used by the wikimedia/selenium config, which is more common.